src/STW.StubApi.Presentation/Services/Bcp/BcpController.cs

```diff
@@ -0,0 +1,43 @@
+namespace STW.StubApi.Presentation.Services.Bcp;
+
+using Microsoft.AspNetCore.Mvc;
+using Models;
+
+[ApiController]
+[Route("bcps")]
+public class BcpController
+{
+    private static readonly List<BcpSearchResultDto.BcpDto> Bcps =
+    [
+        new BcpSearchResultDto.BcpDto
+        {
+            Id = 1,
+            Name = "Belfast Airport",
+            Code = "GBBEL1",
+        },
+        new BcpSearchResultDto.BcpDto
+        {
+            Id = 49,
+            Name = "Manchester Airport",
+            Code = "GBMNC1",
+            Suspended = true,
+        },
+    ];
+
+    [HttpGet("search")]
+    [Consumes("application/json")]
+    public BcpSearchResultDto Search([FromQuery] string code)
+    {
+        return new BcpSearchResultDto
+        {
+            Page = 1,
+            Elements = 25,
```

Comment:
Does this represent the max items we want per page or the number of elements returned?

Reply:
From IPAFFS it looks like max items per page - confusing I know :D